frontend/app/src/main/java/com/chads/vanroomies/RecyclerData.java

```diff
@@ -0,0 +1,28 @@
+package com.chads.vanroomies;
+// Reference: https://www.geeksforgeeks.org/recyclerview-using-gridlayoutmanager-in-android-with-example/
+public class RecyclerData {
+    private String title;
+    private int imgid;
```

Comment:
Can we make this private variable camel case so imgId? And then change the relevant getters and setters accordingly?

Reply:
Will do~